.github/styles/Kedro/words.yml

```
@@ -0,0 +1,52 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+ignorecase: false
+level: warning
+action:
+  name: replace
+swap:
+  # bad: good
+  'acknowledgement': 'acknowledgment'
+  'auto-complete': 'autocomplete'
+  'a number of': 'few|several|many'
+  'and/or': 'and|or|either or'
+  'back end': 'backend'
+  'bear in mind': 'keep in mind'
+  'culprit': 'cause'
```

```suggestion
  'culprit': 'cause'
  'data set': 'dataset'
```
:)

(Also, would apply to the "web site" rule, too--do you need to create a separate rule for the plural?)